.circleci/config.yml

```diff
@@ -128,16 +143,31 @@ workflows:
           filters: { branches: { only: [ master ] } }
           tf_workspace: preproduction
 
+      - apply:
+          name: plan production
+          requires: [ apply preproduction ]
+          filters: { branches: { only: [ master ] } }
+          tf_workspace: production02
+          tf_command: plan
+
+      - apply:
+          name: plan shared-production
+          requires: [ apply preproduction ]
+          filters: { branches: { only: [ master ] } }
+          tf_tier: shared
+          tf_workspace: production
```

Comment:
`production02`?

Reply:
No shared is interestingly in production.. :)

Reply:
Yeah I never got this either 🤷